geodiff/src/geodiff.cpp

```diff
@@ -653,6 +653,81 @@ int GEODIFF_makeCopy( const char *driverSrcName, const char *driverSrcExtraInfo,
   return GEODIFF_SUCCESS;
 }
 
+int GEODIFF_makeCopySqlite( const char *src, const char *dst )
+{
+  if ( !src || !dst )
+  {
+    Logger::instance().error( "NULL arguments to GEODIFF_makeCopySqlite" );
+    return GEODIFF_ERROR;
+  }
+
+  if ( !fileexists( src ) )
+  {
+    Logger::instance().error( "MakeCopySqlite: Source database does not exist: " + std::string( src ) );
+    return GEODIFF_ERROR;
+  }
+
+  // If the destination file already exists, let's replace it. This is for convenience: if the file exists
+  // and it is SQLite database, the backup API would overwrite it, but if the file would not be a valid
+  // SQLite database, it would fail to open. With this check+remove we make sure that any existing file
+  // gets overwritten, regardless of its original content (making the API easier to use for the caller).
+  if ( fileexists( dst ) )
+  {
+    fileremove( dst );
+  }
+
+  sqlite3 *pFrom;
```

Comment:
cannot we use stuff from here? https://github.com/lutraconsulting/geodiff/blob/737972a4fe94893b7540a460728127dd65d27421/geodiff/src/drivers/sqliteutils.h#L25